lib/bundler/audit/scanner.rb

```diff
@@ -68,7 +68,7 @@ def scan(options={},&block)
         ignore = Set[]
         ignore += options[:ignore] if options[:ignore]
 
-        scan_sources(options,&block)
+        scan_sources(options,&block) unless options[:ignore_insecure_sources]
```

Comment:
That was actually my first solution, but wasn't sure if it would be confusing. Can easily do something like this: https://gist.github.com/krigar/a48e505a9bd5f3aa5d8a

I'll change the pull request to adopt that approach instead.